### DISCUSSION TITLE
Automatic CreatedAt and UpdatedAt timestamps

### DISCUSSION THREAD

**Discussion by yousafsabir** [BeanieODM/beanie#516] [Question] [answered]
Is there a way that beanie handles timestamp fields automatically and you don't have to set them manually?

there is a mixin by Radostin that adds timestamps to your beanie Document: https://github.com/roman-right/beanie/discussions/514

but there should be one line configuration in the Settings class inside Document.
thanks

**Comment by roman-right** [comment, marked as answer]
Hi,
In a month or two I plan to add mixins with different methods implemented for the most common cases. I'll add this one there

**Reply by muhammadnasr** [reply]
any updates on this?

**Reply by dakohhh** [reply]
@roman-right , whats up chief, do you have an update now?

All the best, 
Wisdom Dakoh

**Reply by kaminetzky** [reply]
Hi @roman-right! Any news about this feature? Thank you!
